src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter DATE_FORMATTER = (
+            DateTimeFormatter.ofPattern("dd.MM.yyyy")
+    );
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder result = new StringBuilder(
+                "Report for period "
+        ).append(dateFrom).append(" - ").append(dateTo).append("\n");
```

Comment:
Create constant variables for all literals like HEADER, SEPARATOR, etc.
Use System.lineseparator instead of "\n"